### DISCUSSION TITLE
estoy atrasado en todo lo de frontend, como me pogo al dia

### DISCUSSION THREAD

**Discussion by zero2999** [Launch-X-Latam/MisionFrontEnd#471] [Team Posts]
hola, recien me han confirmado que me aceptaron, y deseo ponermen al dia. necesito ayuda de como hacerlo

**Comment by jtaboracastejon** [comment]
Hola! Todas las grabaciones están disponibles en el equipo de Teams debes tener un correo con la invitación para unirte a el

**Comment by crasx1987** [comment]
Buenas noches, como es el nombre del equipo de teams?

**Comment by JMMorph** [comment]
No hay falla, entra al teams, busca en tu correo te debió llegar la invitación, si le dedicas tiempo suficiente sí alcanzas a ponerte al corriente para el hackaton
